app/routes.py

```diff
@@ -1,2 +1,242 @@
-from flask import Blueprint
+import re
+from app import db
+from app.models.customer import Customer
+from datetime import datetime
+from flask import request, Blueprint, make_response, jsonify
+import requests
+import os
+from app.models.video import Video
+from app.models.rental import Rental
 
+
+videos_bp = Blueprint(
+    "videos", __name__, url_prefix="/videos")
+customers_bp = Blueprint(
+    "customers", __name__, url_prefix="/customers")
+rentals_bp = Blueprint(
+    "rentals", __name__, url_prefix="/rentals")
+
+
+# ----------------------------
+# WAVE 1 - CUSTOMER ENDPOINTS
+# ----------------------------
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def customer_index():
+    customers = Customer.query.all()
+    customers_response = [(customer.to_json()) for customer in customers]
+    return make_response(jsonify(customers_response), 200)
+
+
+@customers_bp.route("", methods=["POST"], strict_slashes=False)
+def create_customer():
+    request_body = request.get_json()
+    if "name" in request_body and "postal_code" in request_body and "phone" in request_body:
+        new_customer = Customer(
+            name=request_body["name"],
+            postal_code=request_body["postal_code"],
+            phone=request_body["phone"],
+            registered_at=datetime.now()
+        )
+        db.session.add(new_customer)
+        db.session.commit()
+        customer_response = {"id": new_customer.customer_id}
+        return jsonify(customer_response), 201
+        # return jsonify(new_customer.to_json()), 201
+    return make_response({"details": "You must include a name, postal code, and phone number"}, 400)
+
+
+@customers_bp.route("/<customer_id>", methods=["GET"], strict_slashes=False)
+def get_one_customer(customer_id):
+    customer = Customer.query.get(customer_id)
+    if customer is None:
+        return make_response("Customer does not exist", 404)
```

Comment:
we can combine these actions into one line:

```suggestion
    customer = Customer.query.get_or_404(customer_id)
```